adv_project/settings.py

```diff
@@ -26,7 +27,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config('DEBUG', cast=bool)
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = config('ALLOWED_HOSTS',cast=Csv())
```

Comment:
Glad that you added the hosts to your config file, many students forget to do this and leave the other hosts exposed. Which is not necessarily but could be a security risk so good job.